src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
For those following along, we discussed this offline and will leave as-is for now. :)

I did create a JIRA to track the larger issue of conditional rendering based on ACLs: https://mesosphere.atlassian.net/browse/DCOS-13720